gcovr/gcov_parser.py

```diff
@@ -959,58 +972,72 @@ def _find_excluded_ranges(
     *,
     warnings: _ExclusionRangeWarnings,
     exclude_lines_by_pattern: Optional[str] = None,
+    exclude_branches_by_pattern: Optional[str] = None,
     exclude_pattern_prefix: str,
 ) -> Callable[[int], bool]:
```

Comment:
Not sure about the syntax here:
```suggestion
) -> Tuple(Callable[[int], bool], Callable[[int], bool]):
```

Reply:
Fixed